controllers/hbase_controller.go

```diff
@@ -672,6 +690,7 @@ func (r *HBaseReconciler) statefulSet(hb *hbasev1.HBase,
 
 	// After revision hash calculation, actually update replica count
 	stsSpec.Replicas = pointer.Int32Ptr(ss.Count)
+	stsSpec.Template.ObjectMeta.Annotations = templateMetadataAnnotations
```

Comment:
what is this for?

Reply:
This is for restoring the filtered annotations so the returned statefulset object is contains the original blacklisted key values, post-hash